American-Elections-and-Crypto.md

```diff
@@ -0,0 +1,96 @@
+
+This bill was proposed during the regulatory siege employed by the SEC in 2023. It can be seen as a type of reaction to the Biden administration since it wanted to provide a statutory framework for digital assets and provide clarity, fill regulatory gaps, and foster innovation while providing adequate consumer protections. The republican Patrick McHenry, Chairman of the House Financial Services Committee, commented on the bill:
+
+“Our goal is to strike the appropriate balance between consumer protection and encouraging responsible innovation. This is the product of an unprecedented joint effort between the House Financial Services and Agriculture Committees, which gives us a better shot at striking that balance. I encourage stakeholders and market participants to provide constructive feedback to help us improve our legislation.”
+
+However, before asserting that Trump would lead a crypto-friendly government, there are also pertinent questions that need to be addressed. 
+
+Former President Donald Trump isn't particularly supportive of cryptocurrency becoming a primary form of currency in the United States, prioritizing the U.S. dollar above all else. During an interview with Yahoo Finance in 2021, Trump emphasized the need for caution regarding cryptocurrency, expressing concerns about China's efforts to develop a cryptocurrency aimed at dominating global markets.
+
+“I'm a big fan of our currency and I don't want to have other currencies coming out and hurting or demeaning the dollar in any way. If you look at a monetary system based on the dollar, if you start losing credibility, all of a sudden you're going to lose that strong monetary system, we have to be very careful about that.”
+
+Trump also criticized Bitcoin during a Fox News interview in the same year. Showing once more a vision of protectionism when regarding the American dollar:
+
+"The currency of this world should be the dollar. And I don't think we should have all of the Bitcoins of the world out there. I think they should regulate them very, very high."
+
+The notion that cryptocurrencies should be heavily regulated is not favored by the majority of crypto investors. Given these remarks, it's challenging to predict how Trump would act in relation to crypto. While his involvement in launching digital assets like NFTs and holding crypto may suggest a favorable stance, it's impossible to make a definitive prediction.
```

Comment:
I don’t this this is true. Most real investors want to see good regulations. Most degens, are not interested in regulations and have run campaigns to demonstrate the bad. One reason degens don’t want heavy regulations is because it would mean an end to their pump and dump projects. Most Cypherpunks don’t care either way, due to the fact, if regulations can crush a crypto project it probably wasn’t following Cypherpunk principles. The rest of us want good regulations to remove bad actors and scammers. However the crypto market that emerges after high regulations will be run by the same people currently running Wall Street and real Cypherpunk projects will likely go back to the underground.

Reply:
I really think most of people from Decred community and other crypto really don't like regulation, but i got this vision mainly from twitter and comments on my articles about regulation. For me it is a good thing. I'll let this first phrase out